stratum/proto/CMakeLists.txt

```diff
@@ -1,11 +1,12 @@
 # Builds protobuf object libraries
+#
+# Copyright 2022-2023 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+#
 
 cmake_minimum_required(VERSION 3.5)
 
 find_package(Protobuf REQUIRED)
-if(NOT Protobuf_FOUND)
```

Comment:
I gather this is being removed because we are building and adding dependency, but would it make sense to leave the error this message in case of build errors that might be hard to catch otherwise?

Reply:
I removed the explict check because it's dead code. The REQUIRED parameter on the find_package() command guarantees that it will fail (with a suitable error message) if the Protobuf package is not found.